src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1984,8 +1984,13 @@ void searchContextProviders
 * 0. Preparations
 * 1. Preconditions
 * 2. Get the complete list of entities from mongo
+*
+* RETURN VALUE
```

Comment:
I think it would be a good idea in this case, to keep the same style in the same library (mongoBackend). If the library were using most of the cases 0/1, then I would prefer int.